internal/flink/command_shell.go

```diff
@@ -36,6 +33,11 @@ func (c *command) newShellCommand(prerunner pcmd.PreRunner) *cobra.Command {
 	pcmd.AddEnvironmentFlag(cmd, c.AuthenticatedCLICommand)
 	pcmd.AddContextFlag(cmd, c.CLICommand)
 
+	if featureflags.Manager.BoolVariation("cli.flink.internal", c.Context, config.CliLaunchDarklyClient, true, false) {
+		cmd.Flags().StringSlice("config-key", []string{}, "app option keys for local mode")
+		cmd.Flags().StringSlice("config-value", []string{}, "app option values for local mode")
```

Comment:
https://github.com/spf13/pflag/issues/129 :(

Reply:
yeah was looking for something like that, would be way simpler 😢

Reply:
The linter should catch this, but I'm not sure if it works for flags behind feature flags...
```suggestion
		cmd.Flags().StringSlice("config-key", []string{}, "App option keys for local mode.")
		cmd.Flags().StringSlice("config-value", []string{}, "App option values for local mode.")
```